### ISSUE TITLE
Timezone Support?

### ISSUE THREAD

**Issue by lordson-ajira** [breejs/later#9]
**TL;DR** Would adding timezone support - esp. IANA timezones with DST - without external dependencies - be a worthwhile effort in this library?
If not, would it be possible to add hooks, so that some other library \*cough\* `moment-tz` \*cough\* could do the timezone+DST specific heavy lifting and `later` could just make use of the `Date` objects provided by it

## Current State
From what I see of the code, `later` currently supports UTC or the server's local timezone alone

## Use Case
I'm writing a small service which gets reminder/schedule from users who could potentially be in different time zones.
`later` fits all my requirements (lightweight, human-readable schedule parsing) except for the timezone part.
I'll anyways be using `later` as timezone support isn't a deal-breaker, but might be easier if atleast some hooks were provided so that it would be possible to "opt-in" for timezone support without having to fork the code.

## Considerations
I wouldn't want to support timezones in `later` itself if its only possible by adding dependencies, because no dependencies is a really good selling point for this library
